lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -5,3 +5,22 @@
  */
 
 include_once 'dosomething_global.features.inc';
+
+/**
+ * Implements hook_form_alter().
+ */
+function dosomething_global_form_campaign_node_form_alter(&$form, &$form_state, $form_id) {
+  global $user;
+  // Check if the user is a mexico or brazil admin
+  if (in_array('brazil admin', $user->roles) || in_array('mexico admin', $user->roles)) {
```

Comment:
these should probably be defined as a var at the top as a constant so it's easier to add new roles to this and compare the arrays